gallery/visualtests/nxGlobalSidebar.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+describe('nx-global-sidebar', function() {
+  const sidebarToggle = '.nx-global-sidebar .nx-global-sidebar__toggle',
+  it('looks right', async function() {
+    await browser.url('#/NxGlobalSidebarExample');
```

Comment:
This should be in a `beforeEach` block.  The way you have it, the 'looks right when closed' test will only work if the 'looks right' test runs right before it.  Tests should be independent of one another.

Reply:
> Ok, I didn't find the uses of beforeEach in other tests to be very clear so I hope that this is correct: 3c3645a

That's not wrong but those nested `describe` blocks are a bit redundant.  I'd just have the outer `describe`, the `beforeEach`, and then two tests: "it looks right when open" and "it looks right when closed".

Reply:
> > Ok, I didn't find the uses of beforeEach in other tests to be very clear so I hope that this is correct: [3c3645a](https://github.com/sonatype/sonatype-react-shared-components/commit/3c3645a17ad72b177f93f9bf1b21925e196da596)
> 
> That's not wrong but those nested `describe` blocks are a bit redundant. I'd just have the outer `describe`, the `beforeEach`, and then two tests: "it looks right when open" and "it looks right when closed".

Ok: https://github.com/sonatype/sonatype-react-shared-components/pull/371/commits/1f6450fe74f09d7308f0f02c977ec8caab236669